app/design/frontend/Magento/luma/Magento_Sales/web/css/source/_module.less

```diff
@@ -343,16 +343,24 @@
         }
 
         .product-item-name {
-            display: inline-block;
+            float: left;
+            width: calc(100% - 20px);
```

Comment:
The `calc` operator is used unproperly here.
It won't give the expected result.
Currently, it results with value "80%".

Reply:
thanks